src/main/scala/outwatch/Pipe.scala

```diff
@@ -16,16 +18,16 @@ object Pipe {
     * @tparam T the type parameter of the elements
     * @return the newly created Pipe.
     */
-  def create[T](seeds: T*): IO[Pipe[T, T]] = create[T].map { pipe =>
+  def create[T](seeds: T*)(implicit s: Scheduler): IO[Pipe[T, T]] = create[T].map { pipe =>
```

Comment:
I wonder, since we now have both `IO` and `Task` in our project as dependencies, if we should continue using one or the other, or abstract over them with the type classes found in `cats-effect`, i.e. 
```scala
def create[F[_]: Sync, T](seeds: T*): F[Pipe[T, T]]
```
What do you guys think? :)

On an unrelated note, I also think we could get rid of the varargs here, I don't think it really ever makes sense to prepend items to a stream?

Reply:
1. I'm in favor of abstracting over IO and Task, this would also allow using outwatch without IO Monads again.

2. One seed item is definitely enough for state-management, which is actually what we need for reactive UIs ;) (#96)

@LukaJCB could you please open PRs/Issues for this? I'll merge now.

Reply:
+1 on abstracting over `IO` and `Task`.

Reply:
I created #135 and #136 :)